### PR TITLE
Add synchronised horizontal scroll

### LINKED ISSUE #16: Include support for independent scrolling in side-by-side diffs

### PR DESCRIPTION
**Pull request by mrfyda**
Fixes #16

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -28,9 +28,7 @@
     "url": "git://github.com/rtfpessoa/diff2html.git"
   },
   "main": [
-    "./dist/diff2html-templates.js",
     "./dist/diff2html.js",
-    "./dist/diff2html-ui.js",
```
Comment (rtfpessoa) [inline bower.json]: This file still exists. Unlike the templates, this one should stay.

**Comment by rtfpessoa** [comment]
Please remove the `dist` contents. Those contents are only pushed at release time.

File: bower.json (hunk 1 above)
Comment (mrfyda) [inline bower.json]: > bower                     invalid-meta The "main" field has to contain only 1 file per filetype; found multiple .js files: ["./dist/diff2html-templates.js","./dist/diff2html.js","./dist/diff2html-ui.js"]

Comment (rtfpessoa) [inline bower.json]: Ups, my bad. Thanks for the heads up. 😸

**Comment by rtfpessoa** [comment]
LGTM 👍

**Pull request merged**